doc/tutorials/run_kata_containers.rst

```diff
@@ -19,14 +19,15 @@ Prerequisites
 #. Refer to the :ref:`ACRN supported hardware <hardware>`.
 #. For a default prebuilt ACRN binary in the E2E package, you must have 4
    CPU cores or enable "CPU Hyper-Threading" in order to have 4 CPU threads for 2 CPU cores.
-#. Follow :ref:`these instructions <Ubuntu Service OS>` to set up the ACRN Service VM
+#. Follow the :ref:`rt_industry_ubuntu_setup` to set up the ACRN Service VM
    based on Ubuntu.
 #. This tutorial is validated on the following configurations:
 
-   - ACRN v1.6.1 (tag: acrn-2020w18.4-140000p)
-   - Ubuntu 18.04.4
+   - ACRN v2.0 (branch: ``release_2.0``)
+   - Ubuntu 20.04
+
 #. Kata Containers are only supported for ACRN hypervisors configured for
```

Comment:
Since ACRN supports two scenarios with Kata, "only" can be removed.

Reply:
OK